pip/pip-278.md

```diff
@@ -0,0 +1,142 @@
+# Background knowledge
+
+Apache Pulsar is a distributed messaging system that supports multiple messaging protocols and storage methods. 
+Among them, Pulsar Topic Compaction provides a key-based data retention mechanism that allows you only to keep the most recent message associated with that key to reduce storage space and improve system efficiency.
+
+Another, the Topic Compaction of new load balancer changed the strategy of compaction, it will only to keep the first value of the key.
```

Comment:
Please help add a link of the proposal of new load balancer so that users can go to the proposal to check the details.